pom.xml

```diff
@@ -100,6 +100,16 @@
       <artifactId>vlingo-common</artifactId>
       <version>0.3.4</version>
     </dependency>
+    <dependency>
```

Comment:
The plugin depends on some Micrometer classes, so when the Telemetry class is loaded, it will fail on runtime. To be honest I don't know how to lazy load it (if it's possible).

I was thinking also that may be a good idea to have some default telemetry by default, so when a developer activates it, can at least see all Telemetry information that will eb published through JMX. 

What do you think?

Reply:
RE dynamic loading, if there is a `NoOpTelemetry`, `ConsoleTelemetry`, or `JMXTelemetry` we would have the interfaces in place to dynamically load various specialized implementations, such as the more elaborate micrometer targets. So `interface io.vlingo.actors.Telemetry`. For dynamic loading see available from Java 8 on:

https://docs.oracle.com/javase/tutorial/deployment/jar/jarclassloader.html

Did I answer your question?